org.osgi.test.assertj.cm/src/test/java/org/osgi/test/assertj/cm/test/Test.java

```diff
@@ -0,0 +1,7 @@
+package org.osgi.test.assertj.cm.test;
+
+import org.osgi.test.assertj.cm.configurationevent.ConfigurationEventConditions;
+
+public class Test {
```

Comment:
:-)

Reply:
had some trouble to get the `org.osgi.test.assertj.test.testutil` package from `assertj.framework`. so i removed them. They are now added.